recOrder/tests/conftest.py

```diff
@@ -0,0 +1,38 @@
+import pytest
+from recOrder.cli.settings import TransferFunctionSettings
+from iohub.ngff import open_ome_zarr
+
+@pytest.fixture(scope="function")
+def setup_default_ctf_settings(tmp_path):
```

Comment:
Sorry I wasn't clear:

Since each of these fixtures are just used once, the test cases could just all receive the 'default' fixture and modify the settings in themselves.